io.github.syntalos.yaml

```diff
@@ -0,0 +1,244 @@
+id: io.github.syntalos
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-23.08
+separate-locales: false
+
+command: syntalos
+rename-icon: syntalos
+rename-desktop-file: syntalos.desktop
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --filesystem=host
+  - --socket=system-bus
+  - --socket=session-bus
+  - --allow=devel
+  - --allow=bluetooth
+  - --env=PATH=/app/bin:/usr/bin:/usr/sbin
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0:/usr/lib/extensions/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0
+  - --require-version=1.12.0
+
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+
+add-extensions:
+  org.freedesktop.Platform.GL:
+    directory: lib/GL
+    version: '23.08'
+    subdirectories: true
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+
+modules:
+- name: essential-dirs
+  buildsystem: simple
+  build-commands:
+    - mkdir -p /app/lib/GL
+
+- name: OpenCL
+  buildsystem: cmake-ninja
+  sources:
+    - type: archive
+      url: https://github.com/KhronosGroup/OpenCL-Headers/archive/v2023.12.14/OpenCL-Headers-v2023.12.14.tar.gz
+      sha256: 407d5e109a70ec1b6cd3380ce357c21e3d3651a91caae6d0d8e1719c69a1791d
+
+- name: meson
+  buildsystem: simple
+  build-commands:
+    - python3 setup.py build
+    - python3 setup.py install --prefix=/app --root=/
+  sources:
```

Comment:
Add 

```yaml
cleanup:
- "*"
```

Reply:
It is, actually - there is a module in Syntalos that compiles & runs C++ code and uses Meson.

But we will need to test that further in the Flatpak bundle and see how we want to deal with this module in Flatpak at all, so I'm fine with the cleanup rule until we have figured the C++ compiling out (this will be tricky and I made it a problem of future-me).